src/WidgetRegistry.ts

```diff
@@ -13,6 +13,8 @@ export type WidgetRegistryItem = WidgetBaseConstructor | Promise<WidgetBaseConst
  */
 export const WIDGET_BASE_TYPE = Symbol('Widget Base');
 
+export type RegistryLabel = string | Symbol;
```

Comment:
Yeah I'll move it there, then both the comments below could be addressed with the type.